yarn.lock

```diff
@@ -1439,6 +1439,19 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
+"@firebase/database@0.8.3", "@firebase/database@^0.8.1":
```

Comment:
Hmm.

Reply:
Oops, don't know how these got in. My bad.